src/app/Http/Controllers/ApiController.php

```diff
@@ -126,12 +131,127 @@ public function updateAvailability(Request $request, Location $location) {
     public function armorvax(Request $request) {
         $data = $request->all();
         $data_string = json_encode($data, JSON_PRETTY_PRINT);
-        $path = sprintf('armorvax/%s/u%d_%s.json', config('app.env'), $request->user()->id, date('Y-m-d_his'));
+        $data_collection = collect(json_decode($data_string));
 
-        // Write the full JSON data to a file on S3
-        \Storage::disk('s3')->write($path, $data_string);
+        $web_appointment_type = AppointmentType::firstOrCreate([
+            'name' => 'Web',
+            'short' => 'web'
+        ]);
+        $armorvax_location_source = LocationSource::firstOrCreate([
+            'name' => 'ArmorVax'
+        ]);
+        // Hardcoded -- boo. This is for Local Health Department.
+        $lhd_location_type_id = 3;
```

Comment:
You could do the same as you did above for this one.

$lhd_location_type_id = LocationType::firstOrCreate(['name'=>'Local Health Department','short'=>'d'])->id;